.github/workflows/publish-site.yaml

```diff
@@ -39,10 +39,6 @@ jobs:
       environment_name: pythia
       path_to_notebooks: 'portal'
       build_command: 'make -j4 html'
-      build_from_code_artifact: 'true'
-      code_artifact_name: 'repo-zip'
```

Comment:
This code was necessary for the way the metrics collection is set up - so it's more complicated than deleting these lines. We can discuss an alternate method (the site building is more important than one page being up to date).

Reply:
This code is after that the `automate-metrics` job is completed in the same workflow, could you please help me understand how it is necessary for the metrics collection?

Reply:
You're correct that the `automate-metrics` job is complete, but it's changes won't be reflected in the deploy without building from the artifact `automate-metrics` creates.

`automate-metrics` saves a version of the repository (with an updated metrics file) as a zipped artifact. The `build` job then looks for that zipped artifact instead of the repository before deploying. This method makes it so we don't have to worry about security with bots pushing changes to an ever-changing metrics file (the file is blank on GitHub and is only populated between these steps).

Reply:
Ok, I see it now; thanks for the clarification!